kiwi/filesystem/isofs.py

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-import os
+from distutils.spawn import find_executable
```

Comment:
The used find_executable method seems to be present for the core distutils, I also see extra distutils packages but we don't need this functionality